Spigot-Server-Patches/0387-Add-more-Zombie-API.patch

```diff
@@ -0,0 +1,144 @@
+From 2e8a68b59cf98cee13b07dac58c834021cf02d2b Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Sun, 7 Oct 2018 04:29:59 -0500
+Subject: [PATCH] Add more Zombie API
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityZombie.java b/src/main/java/net/minecraft/server/EntityZombie.java
+index 04d25ad76..a4b186ca1 100644
+--- a/src/main/java/net/minecraft/server/EntityZombie.java
++++ b/src/main/java/net/minecraft/server/EntityZombie.java
+@@ -28,6 +28,7 @@ public class EntityZombie extends EntityMonster {
+     private int bJ;
+     private float bK;
+     private float bL;
++    private boolean shouldBurnInDay = true; // Paper
+ 
+     public EntityZombie(EntityTypes<?> entitytypes, World world) {
+         super(entitytypes, world);
+@@ -76,14 +77,22 @@ public class EntityZombie extends EntityMonster {
+         this.getDataWatcher().register(EntityZombie.bF, Boolean.valueOf(false));
+     }
+ 
++    public boolean isDrowning() { return dG(); } // Paper - OBFHELPER
+     public boolean dG() {
+-        return ((Boolean) this.getDataWatcher().get(EntityZombie.bF)).booleanValue();
++        return this.getDataWatcher().get(EntityZombie.bF); // Paper - unbox
+     }
+ 
++    public void setArmsRaised(boolean raised) { s(raised); } // Paper - OBFHELPER
+     public void s(boolean flag) {
+-        this.getDataWatcher().set(EntityZombie.bE, Boolean.valueOf(flag));
++        this.getDataWatcher().set(EntityZombie.bE, flag); // Paper - unbox
```

Comment:
Is this necessary to make the code compile? Otherwise this looks like unneeded diff that will only make updating harder.

Reply:
Not at all, no. Just through it in because pet peeves ;)